.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,104 @@
+name: Trigger image scanning on pull request and changes to pr
+on:
+  pull_request:
+    types: [opened, edited, synchronize]
+    branches: [main]
+
+jobs:
+  deploy:
+    name: Trigger image scanning
+    runs-on: ubuntu-latest
+    env:
+      PEAK_USER_ID: 8877
```

Comment:
can this be picked from somewhere, instead of harcoding?

Reply:
Updated.